src/Workspaces/Remote/ServiceHub/Host/RemoteWorkspaceManager.cs

```diff
@@ -58,10 +60,50 @@ internal void InitializeAssetSource(IAssetSource assetSource)
         public IAssetSource? TryGetAssetSource()
             => _solutionAssetSource;
 
+        private static ComposableCatalog CreateCatalog(ImmutableArray<Assembly> assemblies)
+        {
+            var resolver = new Resolver(SimpleAssemblyLoader.Instance);
```

Comment:
Why not use `Resolver.DefaultInstance`?

Reply:
I used the one from other examples I had. If `DefaultInstance` works we can use it.